examples/twittermap/web/conf/application.conf

```diff
@@ -117,6 +117,13 @@ tweet.url = "http://twitterMapIndex-search-proxy.herokuapp.com/search/tweets?q=%
 
 # Pointmap parameters
 # How many recent days of the time range predicate of each pointmap query.
-# pointmap.samplingDayRange = 30
+# pointmap.samplingDayRange = 500
```

Comment:
Why is this change needed?

Reply:
Sorry I change this for testing on my local machine. I will change it back.